### PR TITLE
Use Microsoft.SymbolUploader build task for symbol publishing

### PR DESCRIPTION
**Pull request by tmat**
Replaces symbol publishing VSTS tasks with SymbolUploader msbuild task for publishing symbols to symbol servers. This task is used by .NET and Arcade repositories for publishing symbols. It supports publishing both Portable and Windows PDBs, automatically archives all symbols (explicit archiving is not needed anymore) and doesn't suffer from MAX_PATH issues.

Signed build from SymbolUploader branch that successfully publishes symbols during `Vsbuild` step:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=1942163&_a=summary&view=logs

### CODE REVIEW

**Comment by Pilchie** [comment]
Do repo-toolset repos get this for free?

Also tagging @brettfo for F# builds.

**Comment by tmat** [comment]
> Do repo-toolset repos get this for free? 

Yes. They have been using this task for a while.

**Comment by tmat** [comment]
Of course, the repos need to be on the latest version of the toolset and have their YAML definition updated.

**Comment by tmat** [comment]
test windows_debug_vs-integration_prtest  please

**Comment by tmat** [comment]
test ubuntu_16_debug_prtest  please

**Comment by tmat** [comment]
test ubuntu_16_debug_prtest  please

**Comment by tmat** [comment]
@jaredpar @jasonmalinowski @dotnet/roslyn-infrastructure PTAL.

File: build/ToolsetPackages/RoslynToolset.csproj [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <PackageReference Include="Microsoft.Build.Logging.StructuredLogger" Version="$(MicrosoftBuildLoggingStructuredLoggerVersion)" ExcludeAssets="all" />
     <PackageReference Include="Microsoft.NETCore.Platforms" Version="$(MicrosoftNETCorePlatformsVersion)" ExcludeAssets="all" />
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="$(MicrosoftDotNetBuildTasksFeedVersion)" ExcludeAssets="all" />
+    <PackageReference Include="Microsoft.SymbolUploader.Build.Task" Version="$(MicrosoftSymbolUploaderBuildTaskVersion)" ExcludeAssets="all" />
+    <PackageReference Include="Microsoft.Net.RoslynDiagnostics" Version="$(MicrosoftNetRoslynDiagnosticsVersion)" ExcludeAssets="all" />
```
Comment (jaredpar) [inline build/ToolsetPackages/RoslynToolset.csproj]: Why did this second reference need to get added for this change?

**Comment by jaredpar** [comment]
Does this significantly alter our build times or is it mostly neutral?

File: build/ToolsetPackages/RoslynToolset.csproj (hunk 1 above)
Comment (tmat) [reply build/ToolsetPackages/RoslynToolset.csproj]: Looks like a bad merge. Will remove.

**Comment by tmat** [comment]
test this please

**Pull request merged**